### DISCUSSION TITLE
Copilot student account application rejected

### DISCUSSION THREAD

**Discussion by raba13** [community/community#113213] [GitHub Education]
### Select Topic Area

Question

### Body

I applied with my university student email, image of my University ID and every other information in the form on 17/03/2024 (NZ), and awaiting for the Copilot benefits added to my account. What should be my next step please.
Thanks

**Comment by SebastienXia** [comment]
## Hello, @raba13  👋

⏱️ You next step if your been approved by the process is to wait betwenn 5 - 7 days after your approved.

![Capture](https://github.com/community/community/assets/105004912/f376711b-da07-4e4f-ae23-5f8e086f95f6)

`[close the thread and mark as answered if this helped you resolve your issue]`

**Comment by SebastienXia** [comment]
Moreover, you will receive an email when you have benefited from the Education Pack! :)

**Reply by raba13** [reply]
Thanks for the prompt reply.

Unfortunately it does not appear that my application has been approved by the process yet after its initial rejection.

**Reply by realBoltDev** [reply]
If you applied again after correcting details, you'll see approved status if your application gets accepted otherwise rejected. Also it might take a few days for the benefits to actually reflect in your account.

**Reply by raba13** [reply]
It looks like my application is not getting the approved status because in my university ID the photo is on the front and the validity is on the other side of the ID, therefore I put the front and back side image in one page and applied.
Please advise how to resolve the rejection.

**Reply by realBoltDev** [reply]
That should work. A friend of mine did that. His ID was in another language so he wrote a translation on a paper with ID beside it and they accepted it. Maybe try using some other documents which can verify you as a student. The fastest way to get accepted would be to connect the email id provided by your university to your GitHub account and apply using it.

**Reply by raba13** [reply]
My university email is the secondary email,, do not want to flip with the primary email so as not to break and university lab repository connection.

Is there no way for the application to be reviewed by a person?

**Reply by realBoltDev** [reply]
You don't have to change it to primary. There should be an option to choose from your both emails when applying for academic verification. Select university email and proceed.

**Reply by raba13** [reply]
Many thanks for your advise. While going over them I realized that my image format was JPG which I converted to PNG and it seem to have worked.

**Reply by realBoltDev** [reply]
Ok great. Hopefully your application will be accepted now.
``Please close the thread and mark as answered if your issue has been resolved``

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @raba13! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by raba13** [reply]
Many Thanks for your advise and help.

**Comment by raba13** [comment]
Answered.
